### DISCUSSION TITLE
It seems that the `OnRecordListExternalAuths` events are not being fired.

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1042] [Q&A] [answered]
I'm on v0.8.0-rc3

It seems that the `OnRecordListExternalAuths` events are not being fired.  I'm making the Javascript call below.  I'm not sure what is supposed to happen, but it seems like this event should be firing every time that I refresh the browser.  The `OnRecordAuthRequest` is working.

```go
	app.OnRecordAuthRequest().Add(func(e *core.RecordAuthEvent) error {
		log.Println("OnRecordAuthRequest: " + e.Token)
		return nil
	})

	app.OnRecordListExternalAuths().Add(func(e *core.RecordListExternalAuthsEvent) error {
		log.Println("OnRecordListExternalAuths:")
		return nil
	})
```

```javascript
  const authMethods = await client.collection('users').listAuthMethods()
```

**Comment by ganigeorgiev** [comment, marked as answer]
`OnRecordListExternalAuths` fires only on listing the linked external auths for an auth record, aka. when calling:

```
GET /api/collection/:collection/records/:id/external-auths
```

I'll consider naming it better before the final v0.8.0 (at least it should have the `*Request` suffix for consistency and to indicate that this is a `HTTPContext` related event hook).
